Neos.Flow/Classes/Configuration/Source/YamlSource.php

```diff
@@ -53,6 +53,7 @@ public function has($pathAndFilename, $allowSplitSource = false)
     {
         if ($allowSplitSource === true) {
             $pathsAndFileNames = glob($pathAndFilename . '.*.yaml');
+            $pathsAndFileNames = array_merge($pathsAndFileNames, glob($pathAndFilename . '.*.yml'));
```

Comment:
why `array_merge` wouldn't
```php
$pathsAndFileNames = [glob($pathAndFIlename . '.*yaml'), glob($pathAndFilename . '.*.yml')];
```
work the same?

Reply:
Two globs in an array would mean two dimensional array, so that would then need to be treated differently.

Basically [[a,b], [c]] vs. [a, b, c] (array_merge).